CMakeLists.txt

```diff
@@ -101,10 +101,10 @@ else (WIN32)
 endif (WIN32)
 
 # set that name of the main output file as a target name
-if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
-    set(COMMON_CLANG_LIBRARY_NAME "opencl-clang")
+if (NOT DEFINED OPENCL_CLANG_LIBRARY_NAME)
```

Comment:
As we rename this macro.  Do we need to notify IGC compiler team?  Maybe they use "COMMON_CLANG_LIBRARY_NAME" during their compilation.

Reply:
library name in igc build is libopencl-clang.so

Reply:
e.g. xmain 20230302_160000 nightly is using: /rdrive/ref/opencl/runtime/linux/oclgpu/agama-ci-devel-570-EL86/libopencl-clang.so.11